fast64_internal/f3d/f3d_material.py

```diff
@@ -2440,6 +2442,61 @@ def reloadDefaultF3DPresets():
             update_preset_manual_v4(material, presetNameToFilename[material.f3d_mat.presetName])
 
 
+class UpdateColorSpacePopup(Operator):
+    bl_label = "Update Color Space"
+    bl_idname = "dialog.update_colorspace"
+    bl_description = "Update color space"
+    bl_options = {"UNDO"}
+
+    already_invoked = False  # HACK: used to prevent multiple dialogs from popping up
+
+    def invoke(self, context: Context, event: Event):
+        if UpdateColorSpacePopup.already_invoked:
+            return {"FINISHED"}
+        scene = context.scene
+        view_settings = scene.view_settings
+        # Check if color space is correct
+        if not scene.dont_ask_colorspace and (
+            scene.display_settings.display_device != "sRGB"
+            or view_settings.view_transform != "Standard"
+            or view_settings.look != "None"
+            or view_settings.exposure != 0.0
+            or view_settings.gamma != 1.0
+        ):
+            UpdateColorSpacePopup.already_invoked = True
+            return context.window_manager.invoke_props_dialog(self, width=400)
+
+        return {"FINISHED"}
+
+    def draw(self, context: Context):
+        col = self.layout.column()
+        multilineLabel(
+            col,
+            f'The colorspace for the current scene "{context.scene.name}"\nwill cause inaccurate preview compared to N64.\nWould you like to update it?',
```

Comment:
I would like to suggest simplifying this message in the most likely case of there being only a single Scene in the blend file

This would also help not confusing people in oot with blender scene / oot scene

Something like (ugly pseudocode)

```suggestion
            (f'The colorspace for the current scene "{context.scene.name}"' if len(bpy.data.scenes) >= 2 else "The colorspace settings\n") + '\nwill cause inaccurate preview compared to N64.\nWould you like to update it?',
```